src/booleans.js

```diff
@@ -1,61 +1,62 @@
 function negate(a) {
-  // your code here
+  return !a;
 };
 
 function both(a, b) {
-  // your code here
+  return a && (b) || (a && b);
```

Comment:
this just says the same thing twice but with unnecessary parentheses. i.e. can be change to
```
return a && b;
```